project.yaml

```diff
@@ -1315,11 +1343,16 @@ Layer:
     <<: *extents
     Datasource:
       <<: *osm2pgsql
-      table: |2-
-        (SELECT way, place, name, ref
-               FROM planet_osm_point
-               WHERE place IN ('city', 'town') AND (capital = 'yes' OR capital = '4')
-              ) AS placenames_capital
+      table: |-
+        (SELECT
+            way,
+            place,
+            name,
+            ref
+          FROM planet_osm_point
+          WHERE place IN ('city', 'town')
+            AND capital IN ('yes', '4')
```

Comment:
I think we shouldn't include 2. Normally capital=2 is tagged as capital=yes. There are only 2 objects with capital=2 in the database.

We could include capital=3, but it's not urgent as there are only 8 instances in the database.